.tool-versions

```diff
@@ -0,0 +1 @@
+golang 1.17
```

Comment:
Fix the go version.  According to https://github.com/kennyp/asdf-golang#version-selection `asdf` should pick up the version from `go.mod`, but it didn't work for me.  If this is a `me` problem, please let me know and I'll update the docs / remove this file. Thanks!

Reply:
works for me, we used to have this file but because of that same doc you're linking we removed it, make youre your goenv is properly set tho

Reply:
Without this file, asdf-golang doesn't seem to correctly pick up the necessary go version and instead relies on the shimed/system-installed go version.

```
> asdf install
golang 1.16 is already installed

> asdf list golang
  1.16
  1.17

> cat go.mod | head -3
module github.com/mongodb/mongocli

go 1.17
```